app/server.js

```diff
@@ -60,8 +64,11 @@ const dbLogger = getChildLogger({
 
 
     // TODO: Configure actual routes here:
+    // SIDENOTE: Routes can be done with react I believe.
+    // We can create our main architecture through this component
     app.get('/', (req, res) => {
-        res.send('Hello world!');
+        var html = ReactDOMServer.renderToString(React.createElement(Routes));
```

Comment:
`const html =`

Reply:
I'm not sure we can use const here. Perhaps 'let' might be the way to go.

Reply:
I'm pretty sure can use const. const refers to immutability of the reference not the value. 

https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75